_includes/faq.html

```diff
@@ -0,0 +1,18 @@
+
+{% assign list = site.data.faq[page.lang] %}
+{% assign counter = 1 %}
+{% for item in list %}
```

Comment:
Should we add something in case there are no entries in the faq data file?

Reply:
yes, I found blank pages kind of misleading

Reply:
Good, gimme a sec